content/en/tracing/trace_collection/custom_instrumentation/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Custom Instrumentation
-kind: documentation
+kind: multi-code-lang
```

Comment:
Should this be type?

Reply:
Good catch! It's actually not supposed to be a multi-code-lang page anymore, which is why this didn't break anything I guess...